packages/fetchai/skills/tac_control_contract/game.py

```diff
@@ -0,0 +1,935 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
```

Comment:
Relative to the original tac, what did you actually change in this file? Please mark the lines which you changed so I can review